### PR TITLE
Host tags

### PR DESCRIPTION
**Pull request by nicktate**
* Added `GET` for individual host
* Added `UPDATE` for host to add/delete tags

Depends on https://github.com/containership/containership.k8s.translator/pull/9

@normanjoyner

### CODE REVIEW

File: lib/kubernetes-api.js [hunk 1]
```diff
@@ -529,12 +573,77 @@ class KubernetesApi extends ApiImplementation {
...
+        });
+    }
+
     //Look here!
     //Use os module to get info about master!
     getHosts(cb) {
         const{ handler, errorHandler } = this.sanitizeCallback(cb);
-
-        return async.parallel({
+return async.parallel({
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: fix spacing here

File: lib/kubernetes-api.js [hunk 2]
```diff
@@ -529,12 +573,77 @@ class KubernetesApi extends ApiImplementation {
...
+        return this.getHost(host_id, (err, host) => {
+            if (err) {
+                return errorHandler(err);
+            }
+
+            tags = flatten({ tags: tags });
+            const current_tags = flatten({
+                tags: host.tags
+            });
+
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: comment about why we need `label_regex`

File: lib/kubernetes-api.js [hunk 3]
```diff
@@ -322,6 +323,49 @@ class KubernetesApi extends ApiImplementation {
         });
     }
 
+    scaleDownContainers(app_id, count, cb) {
```
Comment (normanjoyner) [inline lib/kubernetes-api.js]: all of these scale related functions should be on `master` already

File: lib/kubernetes-api.js (hunk 2 above)
Comment (nicktate) [inline lib/kubernetes-api.js]: this is updated , just outside of comment range

**Pull request merged**